src/types/clipboard.ts

```diff
@@ -2,6 +2,13 @@ import { CommandResult } from "./commands";
 import { Dimension, HeaderIndex, UID, Zone } from "./misc";
 import { GridRenderingContext } from "./rendering";
 
+export enum ClipboardContentType {
+  PLAIN = "text/plain",
+  HTML = "text/html",
```

Comment:
I think we are using PascalCase for most enums
```suggestion
  PlainText = "text/plain",
  Html = "text/html",
```